nb_lightning_talks

```diff
@@ -40,6 +58,243 @@ html"""
 <iframe width="680" height="677" src="https://www.youtube.com/embed/JxS5E-kZc2s" title="Funny Cats Compilation (Most Popular) Part 1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 """
 
+# ╔═╡ ff96dd3f-6646-41d9-a4b6-ccefbfcbb855
+md"""
+### 🔍 What is UnfoldMakie?
+
+A Julia package for ERP plotting.
+
+Built on top of:
+* **_Unfold.jl_** - Performing rERP analysis
+* **_Makie.jl_** - High-performance visualization for creating interactive, customizable 2D and 3D plots and animations
+* **_AlgebraOfGraphics.jl_** (AoG) - Mapping data to visual elements
+"""
+
+# ╔═╡ 2bc27bcf-3453-4de9-9058-7855af330bcb
+md"""
+### Why UnfoldMakie.jl?
+* ⏩ Fast, declarative plotting
+* 🖌️ Grammar-of-Graphics style mapping
+* 🧩 Easy multi-plot layouts with Makie
+* 🛠️ Highly customizable
+"""
+
+
+# ╔═╡ 648e17da-d1ae-4733-8ac6-6b2102b07e7f
+md"""
+### Other tools on the market
+"""
+
+# ╔═╡ c9bb4057-2d59-4ea6-bbf2-307084818133
+md"""
+🧠 Most EEG researchers use MATLAB tools
+
+🐢💸 But MATLAB can be slow, expensive and not so versitile
+
+"""
+
+# ╔═╡ 2c34cc63-8bb3-4e73-b8c5-d8002bbb317a
+md"""
+
+![](https://raw.githubusercontent.com/s-ccs/workshop_unfold_2025/547bc4cd27b0b88f18349f8223298a9f9de9b18d/img/EEG_tools.jpg)
+"""
+
+# ╔═╡ da755221-0b2c-47ea-94c0-bb90fca4b7a8
+md"""
+### Data and packages
+"""
+
+# ╔═╡ b939c7c7-f55f-408f-83b7-70084406170e
+md"""
+### Fast rendering ⚡
+"""
+
+# ╔═╡ a75eb514-efff-4449-bf73-02dd4e1de70c
+@bind current_time PlutoUI.Slider(1:50, show_value=true, default=4)
+
+# ╔═╡ 47c31314-388a-411e-b808-e2fef4e06d56
+begin
+    ### Example data
+    timestamps = 1:100  # Example time points
+    
+    ### Create a figure
+    
+    ### Create an observable for the data (to bind it to the slider)
+    #dat_obs = Observable(dat_raw[:, 1, 1])  # Initial data slice for the topoplot
+    	
+    ### Plot the initial topoplot
+    plot_topoplot(dat_raw[:, current_time, 1], positions = positions)
+
+    #dat_obs[] = dat_raw[:, current_time, 1]
```

Comment:
you want to demo observables, but you are using CairoMakie?!?

Also observables + pluto don't work that well. I'd maybe just skip this part